pkg/torcx/propagate.go

```diff
@@ -242,7 +243,22 @@ func symlinkUnitAsset(applyCfg *ApplyConfig, unitsDir string, asset string) erro
 				// Do not overwrite previous assets
 				return nil
 			}
-			return os.Symlink(path, hostPath)
+			fpDst, err := os.Create(hostPath)
```

Comment:
nit, create/open without matching closes... I know it's ont a long lived process so doesn't matter too much.